docs/pages/docs/reference/configuration.mdx

```diff
@@ -200,7 +200,7 @@ Defaults to `[]`. Tells `turbo` the set of files to consider when determining if
 Setting this to a list of globs will cause the task to only be rerun when files matching those globs have
 changed. This can be helpful if you want to, for example, skip running tests unless a source file changed.
 
-Specifying `[]` will cause the task to be rerun when any file changes.
+Specifying `[]` will cause the task to be rerun when any version-controlled file in the workspace changes.
```

Comment:
Same comment as above. We shouldn't change this since that is (at least aspirationally) what it should do.

Reply:
https://github.com/vercel/turborepo/pull/1775